packages/vue/src/VueTolgee.ts

```diff
@@ -19,23 +20,28 @@ export const VueTolgee = {
       throw new Error('Tolgee instance not passed in options');
     }
 
-    app.mixin({
-      beforeCreate() {
-        this.$options.__keySubscription = tolgee.onNsUpdate(() => {
-          this.$forceUpdate();
-        });
-      },
-      unmounted() {
-        this.$options.__keySubscription.unsubscribe();
-      },
-      methods: {
+    const createTFunc = () => {
+      return (...props) => {
         // @ts-ignore
-        $t(...props) {
-          // @ts-ignore
-          const params = getTranslateProps(...props);
-          const { ns } = params;
-          this.$options.__keySubscription.subscribeNs(ns);
-          return tolgee.t(params);
+        const params = getTranslateProps(...props);
+        return tolgee.t(params);
+      };
+    };
+
+    const tFunc = ref(createTFunc());
+    tolgee.on('update', () => {
+      tFunc.value = createTFunc();
+    });
+
+    app.mixin({
+      computed: {
+        $t: {
+          get() {
+            return tFunc.value;
+          },
+          set() {
+            // do nothing
+          },
```

Comment:
If we don't need a setter we can do the following:
```
$t() {
  return tFunc.value;
}
```

Reply:
Cool 🙂